signer/src/transaction_coordinator.rs

```diff
@@ -427,6 +489,13 @@ where
         // Construct the transaction package and store it in the database.
         let transaction_package = pending_requests.construct_transactions()?;
 
+        self.construct_and_send_bitcoin_presign_request(
+            bitcoin_chain_tip,
+            &pending_requests,
+            &transaction_package,
+        )
+        .await?;
```

Comment:
The coordinator will return without sending the signing request if we don't receive enough Acks from the signers in the expected timeout